src/controllers/movies.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const fetch = require('node-fetch')
+
+
+const getMovies = (req, res) => {
+    res.sendFile(path.join(__dirname, '..', '..', 'public', 'movies.html'));
+};
+
+const getApiMovies = (req, res) => {
+    // fetch movies from other endpoint
+    // send it to front
+    console.log();
+    const url = `https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=${process.env.API_KEY}`
+    fetch(url)
+        .then(data => data.json())
+        .then(data => res.json(data))
+        .catch(err => console.log(err))
+
+}
+
+
+const postMovies = (req, res) => {
+    const query = req.body.name
+    fetchMovies(query)
```

Comment:
reading the name of this function I expect it to create a new movie object which is not the case, so calling it something like `getMoviesList` or just `getMovies` would make more sense also get the search term in a querystring since here you are applying a search query and it is more convenient to use querystring for this purpose and not sending it in the request body.